.travis.yml

```diff
@@ -8,3 +12,5 @@ cache:
     - ~/.cache
 notifications:
   email: false
+script:
```

Comment:
remove `script` config and it will use default script, which should work https://github.com/stereobooster/react-accessible-tabs/blob/master/package.json#L14